js/analyses/getLinkStatistics.js

```diff
@@ -0,0 +1,61 @@
+/** @module analyses/countLinks */
+
+var getAnchors = require( "../stringProcessing/getAnchorsFromText.js" );
+var findKeywordInUrl = require( "../stringProcessing/findKeywordInUrl.js" );
+var getLinkType = require( "../stringProcessing/getLinkType.js" );
+var checkNofollow = require( "../stringProcessing/checkNofollow.js" );
+
+/**
+ * Checks a text for anchors and returns an object with all linktypes found.
+ *
+ * @param {string} text The text to check for anchors.
+ * @param {string} keyword The keyword to use for matching in anchors.
+ * @param {string} url The url of the page.
+ * @returns {object} The object containing all linktypes.
+ * total: the total number of links found
```

Comment:
This doesn't result in readable documentation, we should make sure it does. See:
![screen shot 2015-12-29 at 09 52 53](https://cloud.githubusercontent.com/assets/584693/12031856/f243aba4-ae11-11e5-8d56-0b45dc49c220.png)